src/mysql.erl

```diff
@@ -145,13 +144,11 @@
                        {via, Module :: atom(), ViaName :: term()},
          NamedStatements :: [{StatementName :: atom(), Statement :: iodata()}].
 start_link(Options) ->
-    GenSrvOpts = [{timeout, proplists:get_value(connect_timeout, Options,
-                                                ?default_connect_timeout)}],
     case proplists:get_value(name, Options) of
         undefined ->
-            gen_server:start_link(mysql_conn, Options, GenSrvOpts);
+            gen_server:start_link(mysql_conn, Options, []);
```

Comment:
`gen_server` doesn't have a default timeout for `start_link`? I hope not. (For `gen_server:call/2`, the default timeout is 5000.)

Reply:
Oh, right, I didn't think of that. So I have just been looking into what `gen_server` and `gen` do there:

[`gen_server:start_link/3,4`](https://github.com/erlang/otp/blob/36fdda7ff23fe8d163283287d950ae903de359d1/lib/stdlib/src/gen_server.erl#L179) call [`gen:start/5,6`](https://github.com/erlang/otp/blob/36fdda7ff23fe8d163283287d950ae903de359d1/lib/stdlib/src/gen.erl#L75), which in turn calls [`gen:do_spawn/5,6`](https://github.com/erlang/otp/blob/36fdda7ff23fe8d163283287d950ae903de359d1/lib/stdlib/src/gen.erl#L92), which then uses [`gen:timeout/1`](https://github.com/erlang/otp/blob/36fdda7ff23fe8d163283287d950ae903de359d1/lib/stdlib/src/gen.erl#L362) to retrieve the timeout from the options, defaulting to `infinity` if it is absent.

Reply:
Great.